openedx_tagging/core/tagging/import_export/parsers.py

```diff
@@ -140,20 +141,21 @@ def _parse_tags(
                         )
                     )
                     has_error = True
+                else:
+                    tag_data[req_field] = tag[req_field]
 
-            tag["index"] = row
+            tag_data["index"] = row
             row += 1
 
             # Skip parse if there is an error
             if has_error:
                 continue
 
-            # Updating any empty optional field to None
+            # Optional fields default to None
             for opt_field in cls.optional_fields:
-                if opt_field in tag and not tag.get(opt_field):
-                    tag[opt_field] = None
+                tag_data[opt_field] = tag.get(opt_field) or None
```

Comment:
@rpenido Yeah, I was preserving the previous behavior with that change, since the CSV parser will send us empty strings for `parent_id`, when what we really want is `None`. 
I think that if we ever need the parser to do something different, then there will be a clear use case, and the code and tests can be modified then.